android-components/components/browser/engine-gecko/src/main/java/mozilla/components/browser/engine/gecko/shopping/GeckoProductAnalysis.kt

```diff
@@ -14,6 +14,7 @@ import mozilla.components.concept.engine.shopping.ProductAnalysis
  * @property grade Reliability grade for the product's reviews
  * @property adjustedRating Product rating adjusted to exclude untrusted reviews
  * @property needsAnalysis Boolean indicating if the analysis is stale
+ * @property pageNotSupported Boolean indicating if the page is supported
```

Comment:
🔍 nit: Since it's a negative boolean, maybe we could mentioned true means page is not supported and false means it's supported.